eoscostarica/eosio-dashboard
# feature missed blocks

### feature missed blocks

### What does this PR do?

- Resolve #24 

### Steps to test

1. make run
2. check hapi logs for missed blocks 
3. check charts in http://localhost:3000/dashboard/network

#### CheckList
